docs/PowerFX/TestEngine.PlaywrightScript.md

```diff
@@ -0,0 +1,44 @@
+# TestEngine.PlaywrightScript
+
+`TestEngine.PlaywrightScript(csxFileName)`
+
+The PlaywrightScript function provides a "no cliffs" extensibility for Test Engine providing the ability to execute CSharp Scripts (*.csx) files inside a Test Engine web provider based test that uses Playwright as web page test framework.
```

Comment:
Move to experimental namespace and add aseembly scanning for the loaded csx to be validating against allowed/denied namespaces